src/App.jsx

```diff
@@ -12,21 +11,10 @@ import { AddItem, Home, Layout, List } from './views';
 
 import { getItemData, streamListItems } from './api';
 import { useStateWithStorage } from './utils';
-import { themeSettings } from './Theme/themeSettings';
+import { useTheme } from './Theme/themeSettings';
 
 export function App() {
-	const prefersDarkMode = useMediaQuery('(prefers-color-scheme: dark)');
-
-	const theme = useMemo(
-		() =>
-			createTheme({
-				palette: {
-					...themeSettings,
-					mode: prefersDarkMode ? 'dark' : 'light',
-				},
-			}),
-		[prefersDarkMode],
-	);
+	const theme = useTheme();
```

Comment:
`./Theme/themeSettings` now exports this hook that creates the theme, so the App file doesn't have to know or care about it.

Reply:
Much cleaner! I like it.